### PR TITLE
Add missing documentation for openapiv2 proto definition

### PR DESCRIPTION
**Pull request by bvwells**
Add missing documentation for openapiv2 proto definition.

### CODE REVIEW

**Comment by bvwells** [comment]
@johanbrandhorst  - this PR adds some missing documentation for open API fields. I've taken the OpenAPI spec definitions for them.

**Comment by bvwells** [comment]
Guess that I will need to regenerate some code before completing. Will push updates.

**Comment by johanbrandhorst** [comment]
Please see https://github.com/grpc-ecosystem/grpc-gateway/blob/master/CONTRIBUTING.md for the regeneration steps

**Comment by bvwells** [comment]
Yep, on it. Just had to setup access to docker.pkg.github.com.

**Comment by bvwells** [comment]
All re-generated now.

File: protoc-gen-swagger/options/openapiv2.proto [hunk 1]
```diff
@@ -94,13 +148,20 @@ message Response {
...
+  // A short description of the application. GFM syntax can be used for rich
+  // text representation.
   string description = 2;
+  // The Terms of Service for the API.
   string terms_of_service = 3;
+  // The contact information for the exposed API.
   Contact contact = 4;
+  // The license information for the exposed API.
   License license = 5;
+  // Required Provides the version of the application API (not to be confused
```
Comment (johanbrandhorst) [inline protoc-gen-swagger/options/openapiv2.proto:163]:
```suggestion
  // Required. Provides the version of the application API (not to be confused
```

Comment (bvwells) [reply protoc-gen-swagger/options/openapiv2.proto:163]: Agreed, I was a little unsure about adding this as obviously there is a difference between what the swagger spec defines as required and what is required for protoc-gen-swagger. I'll remove the required statement. I guess we could document the defaults at some point.

Comment (bvwells) [reply protoc-gen-swagger/options/openapiv2.proto]: It looks like there are a number of other descriptions in the proto definition which look like they have been copied from the open api spec verbatim with 'required'. Do you think we should also remove this?

Comment (johanbrandhorst) [reply protoc-gen-swagger/options/openapiv2.proto]: Haha, yeah, just left a comment to this effect.

Comment (bvwells) [reply protoc-gen-swagger/options/openapiv2.proto]: I've not removed all the other instances of required.

**Comment by johanbrandhorst** [comment]
Thanks a ton, this is great! Could you please cherry pick this against v2?

**Comment by bvwells** [comment]
Added PR #1378. Thanks for the review and merge!

**Pull request merged**